add enable_tcp_reset to azurerm_lb_rule & azurerm_lb_nat_rule (#5373)
terraform-providers/terraform-provider-azurerm: website/docs/r/loadbalancer_nat_rule.html.markdown

* `protocol` - (Required) The transport protocol for the external endpoint. Possible values are `Udp`, `Tcp` or `All`.
* `frontend_port` - (Required) The port for the external endpoint. Port numbers for each Rule must be unique within the Load Balancer. Possible values range between 1 and 65534, inclusive.
* `backend_port` - (Required) The port used for internal connections on the endpoint. Possible values range between 1 and 65535, inclusive.

> * `idle_timeout_in_minutes` - (Optional) Specifies the timeout for the Tcp idle connection. The value can be set between 4 and 30 minutes. The default value is 4 minutes. This element is only used when the protocol is set to Tcp.

Reviewer: ```suggestion
* `idle_timeout_in_minutes` - (Optional) Specifies the idle timeout in minutes for TCP connections. Valid values are between `4` and `30` minutes. Defaults to `4` minutes.
```

* `enable_floating_ip` - (Optional) Enables the Floating IP Capacity, required to configure a SQL AlwaysOn Availability Group.

> * `enable_tcp_reset` - (Optional) Indicates whether tcp reset is disabled or enabled. Default `false`.

Reviewer: we can make this:

```suggestion
* `enable_tcp_reset` - (Optional) Is TCP Reset Enabled for this Load Balancer? Defaults to `false`.
```
